scripts/bundleSize.sh

```diff
@@ -2,8 +2,8 @@
 
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
-min=650
-max=675
+min=663
+max=681
```

Comment:
`latest` is currently 672K for `news` so this PR seemingly adds `4K` and news is now 676K